pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -291,6 +297,73 @@ profiles:
 								BindTimeoutSeconds: 600,
 							},
 						},
+						{
+							Name: "PodTopologySpread",
+							Args: &config.PodTopologySpreadArgs{},
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "empty PodTopologySpread, feature DefaultPodTopologySpread enabled",
+			data: []byte(`
+apiVersion: kubescheduler.config.k8s.io/v1beta1
+kind: KubeSchedulerConfiguration
+profiles:
+- pluginConfig:
+  - name: PodTopologySpread
+    args:
+      defaultConstraints:
+`),
+			feature: features.DefaultPodTopologySpread,
+			wantProfiles: []config.KubeSchedulerProfile{
+				{
+					SchedulerName: "default-scheduler",
+					PluginConfig: []config.PluginConfig{
+						{
+							Name: "PodTopologySpread",
+							Args: &config.PodTopologySpreadArgs{
+								DefaultConstraints: []corev1.TopologySpreadConstraint{
+									{
+										MaxSkew:           3,
+										TopologyKey:       corev1.LabelHostname,
+										WhenUnsatisfiable: corev1.ScheduleAnyway,
+									},
+									{
+										MaxSkew:           5,
+										TopologyKey:       corev1.LabelZoneFailureDomainStable,
+										WhenUnsatisfiable: corev1.ScheduleAnyway,
+									},
```

Comment:
nit: reuse the default value defined in `pkg/scheduler/apis/config/v1beta1/defaults.go`?

Reply:
for the whole array? It's not exported, and I think it's best to keep it like that.